cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,16 @@
+import {UploadPage} from "../page/index";
+
+describe("Upload/Download Test", () =>{
+  let upload: UploadPage;
+  const assertMsg = "File Uploaded!";
+
+  before(() => {
+    upload = new UploadPage();
+  });
+
+  it("Uploading a File", () => {
+    upload.visitUploadPage();
+    upload.uploadFile();
```

Comment:
you should receive the fileName of the file to upload as parameter of the method, so in that case the upload page object can be used to upload any file